app/presenters/concerns/analytics_presenter.rb

```diff
@@ -12,26 +12,19 @@ def ids_array(ids)
   def timestamped_pageviews_by_ids(ids)
     # we want the output here to be structured for Flot as used in Hyrax, with unixtimes in milliseconds
     ids = ids_array(ids)
-    start_time = date_uploaded.strftime('%Q').to_i
-    pageviews = Rails.cache.read('ga_pageviews')
-    if pageviews.nil?
-      @pageviews ||= '?'
-      return {}
-    end
-
     data = {}
     count = 0
-    if pageviews.is_a? Hash
-      ids.each do |id|
-        next if pageviews[id].blank?
-        pageviews[id].each do |date, views|
-          timestamp = Date.strptime(date, '%Y%m%d').strftime('%Q').to_i
-          next unless timestamp >= start_time
```

Comment:
We used to do this, to make sure not to include any GA stats that happened before the Work/FileSet date_uploaded. But why? How could there be any usages stats from before a work/file_set was created? Anyway, I stopped doing this check in this new version. I did leave date_uploaded in the #flot_pageviews_over_time method because it's needed for the graph.

Reply:
Yeah I remember thinking that over a couple years ago. NOID reuse is one thing I guess. Maybe migration from another app instance. Stuff like that. Doesn't apply to us anyway.